package.json

```diff
@@ -10,6 +10,12 @@
     "doc": "doc",
     "test": "tests"
   },
+  "browserslist": [
+    "defaults",
+    "Firefox ESR",
+    "Safari > 10",
+    "not ie <= 11"
+  ],
```

Comment:
Best check if this is what we want to support

Reply:
Could we make this config so that it is shared between here and [here](https://github.com/kanselarij-vlaanderen/kaleidos-frontend/blob/development/config/targets.js#L3) (also browserlist format). More info in [cli-docs](https://cli.emberjs.com/release/advanced-use/build-targets/)

Reply:
We supporten volgens mij ook geen IE in het geheel, volgens https://kanselarij.atlassian.net/browse/KAS-1289 hebben we slechts 4 browsers die we bekeken hebben.

Reply:
Lijkt me moeilijk te combineren, mss een comment/referentie naar hier in config/targets.js?